### PR TITLE
Feature/add ods contribution quickstarters & amend all quickstarters to new quickstarter stages

### LINKED ISSUE #261: New Quickstarters for ODS component contributions
**Is your feature request related to a problem? Please describe.**
follow up from opendevstack/ods-core#493 and opendevstack/ods-jenkins-shared-library#284

new quickstarters for 
1. ods-provision-app
1. document-generation-service

### PR DESCRIPTION
**Pull request by clemensutschig**
closes #261

### CODE REVIEW

**Comment by clemensutschig** [comment]
I took the liberty to refactor and change all quickstarters to the new layout - and stages .. which was not done earler

File: e2e-spock-geb/Jenkinsfile [hunk 1]
```diff
@@ -1,65 +1,17 @@
...
-def gitHost
-// vars from jenkins master
-node {
-  dockerRegistry = env.DOCKER_REGISTRY
-  gitHost =  env.BITBUCKET_HOST.split(":")[0]
-}
+library("ods-jenkins-shared-library@${odsGitRef}")
+
+odsQuickstarterPipeline(
+  imageStreamTag: "cd/jenkins-slave-base:${odsImageTag}",
```
Comment (michaelsauter) [inline e2e-spock-geb/Jenkinsfile:7]: Wrong slave image, should be `"cd/jenkins-slave-maven:${odsImageTag}"`.

File: fe-vue/Jenkinsfile [hunk 2]
```diff
@@ -1,76 +1,20 @@
...
-def gitHost
-// vars from jenkins master
-node {
-  dockerRegistry = env.DOCKER_REGISTRY
-  gitHost =  env.BITBUCKET_HOST.split(":")[0]
-}
+library("ods-jenkins-shared-library@${odsGitRef}")
+
+odsQuickstarterPipeline(
+  imageStreamTag: "cd/jenkins-slave-base:${odsImageTag}",
```
Comment (michaelsauter) [inline fe-vue/Jenkinsfile:7]: Wrong slave image, should be `"cd/jenkins-slave-nodejs10-angular:${odsImageTag}"`.

Comment (clemensutschig) [reply fe-vue/Jenkinsfile:7]: not needed @michaelsauter  - this is purely copying files ...

File: e2e-spock-geb/Jenkinsfile (hunk 1 above)
Comment (clemensutschig) [reply e2e-spock-geb/Jenkinsfile:7]: same here ...  (as below) this is purely copying files

File: be-scala-akka/Jenkinsfile [hunk 3]
```diff
@@ -1,75 +1,22 @@
-// build params
-def odsImageTag = env.ODS_IMAGE_TAG ?: 'latest'
 def odsGitRef = env.ODS_GIT_REF ?: 'production'
```
[the remark this answers is not in the document]
Comment (clemensutschig) [reply be-scala-akka/Jenkinsfile:1]: @oalyman  - play is broken currently .. only remove when this works - merge as is :)

**Pull request merged**